src/main/java/com/ays/location/model/Location.java

```diff
@@ -0,0 +1,19 @@
+package com.ays.location.model;
+
+import com.ays.common.model.BaseDomainModel;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.SuperBuilder;
+import org.locationtech.jts.geom.Point;
+
+/**
+ * Location Domain Model to perform data transfer from service layer to controller
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder
+public class Location extends BaseDomainModel {
```

Comment:
Bu lokasyon kısmı doğrudan User'a ait bir obje olduğu için `user.location` paketi altına taşıyıp, adını da `UserLocation` olarak adlandırmamız daha doğru olur. Ek olarak bu obje içerisinde `User` objesinin de olması gerekiyor.

Reply:
`location` package altında olması daha iyi olur. Diğer türlü herşeyi `admin` ve `user` ana package'lerin altına alacağız.